src/index.css

```diff
@@ -95,6 +95,9 @@ body {
   .hover-secondary:hover {
     color: var(--secondary-light-active-transparent);
   }
+  .hover-button-scroll:hover {
+    filter: drop-shadow(0rem 0rem 4rem var(--primary));
```

Comment:
I think that we can use box-shadow here to save a little speed:
https://stackoverflow.com/questions/27212782/box-shadow-circle

Reply:
But it's up to you, this element shouldn't be that hard to computate

Reply:
Not use it anymore